code/modules/medical/genetics/geneticsResearch.dm

```diff
@@ -345,25 +347,13 @@ var/datum/geneticsResearchManager/genResearch = new()
 	tier = 2
 	requiredResearch = list(/datum/geneticsResearchEntry/checker)
 
-/datum/geneticsResearchEntry/rad_dampers
-	name = "Radiation Dampeners"
-	desc = "Reduces the amount of harmful radiation caused by Radiation Emitters."
-	researchTime = 1800
-	researchCost = 80
-	tier = 2
-	requiredResearch = list(/datum/geneticsResearchEntry/rademitter)
-
-	onFinish()
-		genResearch.emitter_radiation -= 45
-		..()
-
 /datum/geneticsResearchEntry/rad_coolant
 	name = "Emitter Coolant System"
 	desc = "Reduces the amount of time required for Radiation Emitters to cool down."
 	researchTime = 1800
 	researchCost = 120
 	tier = 2
-	requiredResearch = list(/datum/geneticsResearchEntry/rademitter)
+	requiredResearch = list(/datum/geneticsResearchEntry/rad_dampers)
```

Comment:
I considered it, but I figure having some progression is still nice. Making research even faster means letting Genetics "max out" even sooner in long rounds, and this probably keeps a little more balance in the Force, since they're getting a free unlock now.